autocompleter/settings.py

```diff
@@ -41,12 +61,3 @@
 # This will obviously not work if MAX_EXACT_MATCH_WORDS == 0 for your install or your provider.
 # Note: AC/Provider and Provider override possible
 MOVE_EXACT_MATCHES_TO_TOP = getattr(settings, 'AUTOCOMPLETER_MOVE_EXACT_MATCHES_TO_TOP', False)
```

Comment:
so... if you look at ycharts
```
(ycharts_systems) Aras-MacBook-Pro:ycharts ara$ git grep MOVE_EXACT_MATCHES_TO_TOP
apps/calculations/autocompleters.py:217:    'MOVE_EXACT_MATCHES_TO_TOP': True,
apps/securities/autocompleters.py:87:    'MOVE_EXACT_MATCHES_TO_TOP': True,
ycharts/settings/ycharts.py:291:AUTOCOMPLETER_MOVE_EXACT_MATCHES_TO_TOP = True
```

we set MOVE_EXACT_MATCHES_TO_TOP at the ac_provider level, but if you look at the code
```
            # We merge exact matches with base matches by moving them to
            # the head of the results
            if settings.MOVE_EXACT_MATCHES_TO_TOP:
                exact_ids = results.pop(0)
```
it is a global setting!

this makes sense because the way the pipeline is set up, we can't do exact matches for some providers and not others ....

Anyway though, now that we have ac level settings this should be an AC setting and we should get it using get_ac_setting and so on...